internal/clients/http/image_builder/image_client.go

```diff
@@ -105,9 +105,26 @@ func (c *ibClient) GetGCPImageName(ctx context.Context, composeID string) (strin
 }
 
 func (c *ibClient) fetchImageStatus(ctx context.Context, composeID string) (*UploadStatus, error) {
+	ctx, span := otel.Tracer(TraceName).Start(ctx, "fetchImageStatus")
+	defer span.End()
 	logger := logger(ctx)
 	logger.Trace().Msgf("Fetching image status %v", composeID)
 
+	composeResp, err := c.checkCompose(ctx, composeID)
+	if err != nil {
+		cloneResp, err := c.checkClone(ctx, composeID)
+		if err != nil {
+			return nil, fmt.Errorf("could not find image neither in compose nor in clones %w", err)
```

Comment:
Mind adding colon, error could be misleading without it:

```suggestion
			return nil, fmt.Errorf("could not find image neither in compose nor in clones: %w", err)
```